src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+
+public class Constants {
+    
+    public static final class Elevator {
+        //Height for each level - meters
+        public static final double HIGH_HEIGHT = 2.286; //Max height is 96.5in so 90in for high ig
+        public static final double MIDDLE_HEIGHT = 1.397; // Middle point between high and low
+        public static final double LOW_HEIGHT = 0.508; //Min height is 12in so 20in for low ig
+        
+        //CAN IDs 
+        public static final int LEFT_MOTOR_CANID = 5; 
+        public static final int RIGHT_MOTOR_CANID = 6; 
+
+        //PID Controller 
+        public static final double ELEVATOR_P = 0.8; 
+        public static final double ELEVATOR_I = 0.0; 
+        public static final double ELEVATOR_D = 0.02; 
+
+        //Speeds - Percentage of max speed
+        public static final double IDLE_SPEED = 0; 
```

Comment:
constants that will always be 0 shouldn't change

Reply:
ngl im kinda slow I didn't even say what to do here, basically don't have constants that will always be 0